documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -66,7 +66,7 @@ spec:
 EOF
 ----
 <1> Allowed values are `ovirt` and `vsphere`.
-<2> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere or `https://<{rhv-short}_engine_host>/ovirt-engine/api/` for {rhv-short}.
+<2> s, `https://<vCenter_host>/sdk` for vSphere or `https://<engine_host>/ovirt-engine/api/` for {rhv-short}.
```

Comment:
@RichardHoch Was this changed by mistake?

Reply:
@mguetta1 Must have been -- was just looking at that now.  Fixed.